docs/Text.md

```diff
@@ -0,0 +1,185 @@
+| [`.cell()`](#cell)  | one | yes | no | yes | Inserts a single-line text string within the boundaries of a given box, optionally with background and border. |
+| [`.multi_cell()`](#multi_cell) | several | yes | no | yes | Inserts a multi-line text string within the boundaries of a given box, optionally with background and border. |
+| [`.write()`](#write) | several | no | no | auto | Inserts a multi-line text string within the boundaries of the page margins, starting at the current x/y location (typically the end of the last inserted text). |
+| [`.write_html()`](#write_html) | several | no | yes | auto | From [html.py](HTML.html). An extension to `.write()`, with additional parsing of basic HTML tags.
+
+## Typographical Limitations
+
+There are a few advanced typesetting features that fpdf doesn't currently support.
+* Automatic ligatures - Some writing systems (eg. most hindic scripts such as Devaganari, Hangul) frequently combine a number of written characters into a single glyph. This would require advanced font analysis capabilities, which aren't currently implemented.
+* Contextual forms - In some writing systems (eg. Arabic, Hebrew, Mongolian, etc.), characters may take a different shape, depending on whether they appear at the beginning, in the middle, or at the end of a word, or isolated. Fpdf will always use the same standard shape in those cases.
+* Vertical writing - Some writing systems are meant to be written vertically. Doing so is not directly supported. In cases where this just means to stack characters on top of each other (eg. Chinese, Japanese, etc.), client software can implement this by placing each character individuall at the correct location. In cases where the characters are connected with each other (eg. Mongolian), this may be more difficult, if possible at all.
+
+## Text Formatting
+For all text insertion methods, the relevant font related properties (eg. font/style and foreground/background color) must be set before invoking them. This includes using:
+* [`.set_font()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_font)
+* [`.set_text_color()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_text_color)
+* [`.set_draw_color()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_draw_color) - for cell borders
+* [`.set_fill_color()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_fill_color) - for the background
+
+In addition, some of the methods can optionally use [markdown](TextStyling.html#markdowntrue) or [HTML](HTML.html) markup in the supplied text in order to change the font style (bold/italic/underline) of parts of the output.
+
+## Change in current position
+`.cell()` and `.multi_cell()` let you specify where the current position (`.x`/`.y`) should go after the call. This is handled by the parameters `new_x` and `new_y`. Their values are one out of the Enums `.XPos` and `.YPos` respectively, which offer the following options:
+
+#### .XPos
+* XPos.LEFT    - left end of the cell
```

Comment:
Could you please add backquotes or double asterisks around those constant?

Also, wouldn't it be better to just point to the generated documentation in https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.XPos ? This way the fields descriptions will stay up-to-date if the code changes, no risk for this doc paragraph to become outdated.

Reply:
Highlighting them is a good idea.

Not sure about replacing them with links here.
(Yes, all documentation needs updating on API changes.)